### ISSUE TITLE
CrossMediaManager.Current.Init() turns off audio in iOS

### ISSUE THREAD

**Issue by dczaretsky** [Baseflow/XamarinMediaManager#643]
## 🐛 Bug Report

I've including the following line in the AppDelegate.cs file, as described in the setup:
CrossMediaManager.Current.Init();

However, this line seems to take control of the audio player and turns off current playing audio when the app is opened without playing anything. This is not the expected behavior, since the current audio should only stop when you hit play.

### Expected behavior
App should open and continue playing current audio from another player.

### Reproduction steps
1. Include **CrossMediaManager.Current.Init();** in the FinishedLaunching() function in AppDelegates.cs.

2. Play a song from iTunes or any other audio player.

3. Open your app. On startup the current song will stop playing.

### Configuration

**Version:** 1.x

**Platform:** 
- [ X] :iphone: iOS
- [ ] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by OdetteP** [comment]
This is not possible. You cannot play an audio file in one app and open another an expect it to play in the other app.

**Comment by dczaretsky** [comment]
I'm not sure why this bug was closed. In most audio / music apps, opening the app does not turn off what is currently being played UNTIL you actual hit the play button in the app. That is not the case here.

After further investigation, I found that disabling the **CrossMediaManager.Current.Init()** in AppDelegate.cs prevents this behavior from happening. Therefore, it seems like something in that Init() function is taking hold of the player, which should not happen until something is played.
